Dynatrace/dynatrace-operator
# Close issue ticket automatically if "support/feature request" label found

## Description

The issue is automatically closed if labeled as `Feature request` or `Support request`:
- appropriate comment is added
- the ticket is closed

The issue is re-open if `Feature request` or `Support request` label is removed:
- appropriate message about the mistake is added
- the ticket is re-open

## How can this be tested?
Add, remove "feature request" and "support request" labels. Check if  comments are added and the issue is closed or re-open (consider forked repo).

## Checklist

~~- [ ] Unit tests have been updated/added~~
